src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -24,6 +24,8 @@ internal class SimpleResolver : INameResolver
     {
         protected TexlFunction[] _library;
         protected EnumSymbol[] _enums = new EnumSymbol[] { };
+
+        // $$$ This isn't used anymore, but still required by INameResolver ...
```

Comment:
I mean, this can be cleaned up, and change the Document prop to return null at least

Reply:
upon further review, derived classes are setting ...

Reply:
Is this used at all in the base class?

Reply:
yes, sql uses it.

Reply:
Sql is a derived class. Is it needed in the base class or can this be moved to the derived classes?

Reply:
It's used by the INameResolver interface that this type implements, so it can't be moved to a derived class - we'd need to remove it from the itnerface.  (interface is like a base class here)